microhapulator/cli/dist.py

```diff
@@ -15,8 +15,13 @@ def subparser(subparsers):
         'default, output is written to the terminal (standard output)'
     )
     cli.add_argument(
-        'gt1', help='genotype data (in JSON format) for individual 1'
+        '-p', '--ploidy', type=int, metavar='PLD', default=2,
+        help='for simulated genotypes, the number of distinct haplotypes '
+        'present in the sample; default is 2 (for a single diploid genotype)'
```

Comment:
It should be possible to infer the ploidy automatically when populating `SimulatedGenotype` objects from BED data. I don't like having to state this explicitly, but we'll leave it in for now.